transport-security.md

```diff
@@ -13,16 +13,18 @@ Note: security controls for signing and encrypting of application level messages
 ## Transport security
 One should secure all APIs assuming they can be accessed from any location on the internet. Information MUST be exchanged over TLS-based secured connections. No exceptions, so everywhere and always. This is [required by law](https://wetten.overheid.nl/BWBR0048156/2023-07-01). One SHOULD follow [the latest NCSC guidelines for TLS](https://english.ncsc.nl/publications/publications/2021/january/19/it-security-guidelines-for-transport-layer-security-2.1)
 
-<div class="rule" id="api-11">
-  <p class="rulelab"><strong>API-11</strong>: Secure connections using TLS</p>
+<span id="api-11"></span>
+<div class="rule" id="/transport/TLS">
+  <p class="rulelab"><strong>/transport/TLS</strong>: Secure connections using TLS</p>
```

Comment:
De standaard voor rule-IDs is `kebab-case`. Deze zouden dus altijd lowercase moeten zijn.